### DISCUSSION TITLE
Assistance for Deleting Outdated Content

### DISCUSSION THREAD

**Discussion by Venkatesh-Dayani23** [Sunbird-Ed/Community#673] [Bugs]
Hi Team,

We are currently using Sunbird version 5.0.0 and require assistance in the process of removing outdated content from databases. Could you please provide us with instructions on the deletion process.

cc: @raghupathiguduri

**Comment by vinukumar-vs** [comment]
I would suggest to move this discussion to Knowlg Building Block. As it is related to content, it will be better to discuss in the knowlg BB discussions.

Make sure you add the details of how you decide which content is outdated?

@mohitga [@pallakartheekreddy](https://github.com/pallakartheekreddy)  [@maheshkumargangula](https://github.com/maheshkumargangula)

**Reply by Venkatesh-Dayani23** [reply]
Thanks for update @vinukumar-vs
